bgpd/bgp_errors.c

```diff
@@ -16,54 +16,73 @@ static struct log_ref ferr_bgp_warn[] = {
 		.code = EC_BGP_ASPATH_FEWER_HOPS,
 		.title = "BGP AS-path conversion has failed",
 		.description = "BGP has attempted to convert a AS2 to AS4 path and has failed",
-		.suggestion = "Open an Issue with all relevant log files and restart FRR"
+		.suggestion = "Open an Issue with all relevant log files and restart FRR",
 	},
 	{
 		.code = EC_BGP_DEFUNCT_SNPA_LEN,
 		.title = "BGP has received a value in a reserved field",
-		.description = "BGP has received a non-zero value in a reserved field that was used for SNPA-length at one point in time",
-		.suggestion = "BGP has peered with either a router that is attempting to send SNPA data or it has received a corrupted packet.  If we are peering with a SNPA aware router(unlikely) upgrade that router, else open an Issue after gathering relevant log files",
+		.description = "BGP has received a non-zero value in a reserved field that was used for "
```

Comment:
is there a benefit to these white-space changes? there seem to be many many lines that have just be re-wrapped.

Reply:
Fixed it